plugin/documents.py

```diff
@@ -483,9 +483,22 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'markup.changed'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
+            annotations_template = """
+            <body id="lsp-line-annotation">
+                <style>
+                    html, body {{
+                        font-family: system;
+                        margin: 0;
+                        padding: 0;
+                    }}
```

Comment:
This creates this weird behavior on hovering where the bounding box gets bigger:
https://user-images.githubusercontent.com/153197/118176061-34f80a00-b431-11eb-8123-9e664782b84b.mov

It doesn't happen by default.